Makefile

```diff
@@ -104,7 +104,16 @@ ifneq ($(scripts),)
 endif
 
 .PHONY: test
-test: test-race test-runs test-sessions
+test: test-race test-runs test-sessions test-opa
+
+.PHONY: test-opa
+test-opa:
```

Comment:
Doesn't this require fixing in the CI too?

Reply:
if opa is not installed, it would do nothing... will check how to get opa installed though.

Reply:
added to ci